frontend/src/docs/DocsWrapper.spec.tsx

```diff
@@ -1,13 +1,48 @@
 import { ThemeProvider } from '@mui/material/styles'
 import { render, screen, waitFor } from '@testing-library/react'
+import { useGetUiConfig } from 'actions/uiConfig'
+import { useGetCurrentUser } from 'actions/user'
 import mockRouter from 'next-router-mock'
-import { describe, expect, it } from 'vitest'
+import { SideNavigationProps } from 'src/wrapper/SideNavigation'
+import { TopNavigationProps } from 'src/wrapper/TopNavigation'
+import { describe, expect, it, vi } from 'vitest'
 
 import { lightTheme } from '../theme'
 import DocsWrapper from './DocsWrapper'
 
+vi.mock('actions/uiConfig', () => ({
+  useGetUiConfig: vi.fn(),
+}))
+
+vi.mock('actions/user', () => ({
+  useGetCurrentUser: vi.fn(),
+}))
+
+vi.mock('src/wrapper/TopNavigation', () => ({ default: (_props: TopNavigationProps) => <></> }))
```

Comment:
We should be mocking the Wrapper instead of these components. These tests should only be testing DocsWrapper.

Reply:
Because we're back to using the old wrapper I've just reverted the changes for this file.